src/avtk/clipselector.cxx

```diff
@@ -232,7 +277,49 @@ void ClipSelector::resize(int X, int Y, int W, int H)
 	redraw();
 }
 
+void setRecordBarsCb(Fl_Widget *w, void* data)
+{
+    if(!w || !data)
+        return;
+
+	ClipSelector *track = (ClipSelector*)w;
+	long bars = (long)data;
+	if(bars == -2){        
+		const char* answer = fl_input("Enter a custom number: ");
+        bars = atoi(answer);
```

Comment:
error check *before* calling `atoi()` here.

Reply:
sorry. that was a quick hack that i forgot